src/main/webapp/WEB-INF/view/activityfeed.jsp

```diff
@@ -47,18 +47,20 @@ List<Event> events = (List<Event>) request.getAttribute("events");
           }
           else {
             String conversationTitle = information.get(1);
-              if (event.getType().equals("Conversation")) {
+              if (event.getType().equals("Conversation") && information.size() > 2) {
+                String conversationId = information.get(2);
               %>
                 <a href="/profile/<%= userName %>"><%= userName %></a> created a new conversation: 
-                <a href="/chat/<%= conversationTitle %>"><%= conversationTitle %></a>
+                <a href="/chat/<%= conversationId %>"><%= conversationTitle %></a>
           </li>
               <%
               }
-              else if (event.getType().equals("Message")) {
+              else if (event.getType().equals("Message") && information.size() > 3) {
```

Comment:
Can we ever have the first part of the condition without the second? Are you thinking about backward compatibility (handling already existing data)?

Reply:
Yeah, I didn't want it to throw exceptions on the already existing events.